README.md

```diff
@@ -906,7 +906,7 @@ See https://github.com/n1trux/awesome-sysadmin#monitoring and https://github.com
   * [Lychee](http://lychee.electerious.com/) - An open source grid and album based photo-management-system. ([Source Code](https://github.com/electerious/Lychee)) `MIT` `PHP`
   * [MediaDrop](http://mediadrop.net/) - A video, audio and podcast publication platform. ([Source Code](https://github.com/mediadrop/mediadrop)) `GPLv3` `Python`
   * [Mediagoblin](http://mediagoblin.org) - A Free software media publishing platform that anyone can run. You can think of it as a decentralized alternative to Flickr, YouTube, SoundCloud, etc. `AGPLv3` `Python`
-  * [MinigalNano](https://github.com/sebsauvage/MinigalNano) - MinigalNano is a very simple image gallery. It adheres to the KISS principle and is very easy to install. MinigalNano does not have a web admin interface: You just upload your images in the photo folder on your server (using FTP, SFTP). ([Demo](http://boards.tomcanac.com/)) `GPLv3/CCBYNCNDv4` `PHP`
+  * [MinigalNano](https://github.com/sebsauvage/MinigalNano) - MinigalNano is a very simple image gallery. It adheres to the KISS principle and is very easy to install. MinigalNano does not have a web admin interface: You just upload your images in the photo folder on your server (using FTP, SFTP). ([Demo](www.minigal.dk/minigalnano/)) `AGPLv3` `PHP`
```

Comment:
www.minigal.dk/minigalnano is not the same project, sebsauvage/MinigalNano is based on it but the dk version is ancient; it is on 0.3.5, 0.3.7 was released [23 mar 2013](https://github.com/sebsauvage/MinigalNano/commit/c6b911cc2947245479b475d96dccd091902ab5c3), so at least 4 years old.

I was not aware the minigalnano demo was down/removed from the README :/ I think you can simply remove the link.